nixd/include/nixd/Controller/Configuration.h

```diff
@@ -13,7 +13,7 @@ namespace nixd {
 // NOLINTBEGIN(readability-identifier-naming)
 struct Configuration {
   struct Formatting {
-    std::vector<std::string> command = {"nixpkgs-fmt"};
+    std::vector<std::string> command = {"nixfmt"};
```

Comment:
Actually only this line will be sufficient if you want to change the default. Some regressions tests might be failing because cannot find *nixfmt*. Maybe wait for #570